php/WP_CLI/Dispatcher/Subcommand.php

```diff
@@ -506,7 +506,7 @@ public function invoke( $args, $assoc_args, $extra_args ) {
 	 * Get an array of parameter names, by merging the command-specific and the
 	 * global parameters.
 	 *
-	 * @param array  $spec Optional. Specification of the current command.
+	 * @param array $spec Optional. Specification of the current command.
```

Comment:
This looks to be `array{name:array<string>}` as its shape. Obviously, the `$spec` array probably has much more, but that's the part this function needs.

The return type looks to be `string[]`.

Reply:
Thanks @BrianHenryIE 

Extra space has been removed. It is phpcs fix